spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -49,6 +49,10 @@
 
 	private ConfigurableApplicationContext parentContext;
 
+	public AggregateApplicationBuilder(String... args) {
```

Comment:
This constructor would also function as a no-arg constructor if we don't pass any `String` arguments.